docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,62 @@
+# Manually Add Calico Networking to Mesos
+This guide will walk you through manually adding Calico networking to your Mesos deployment. You must follow these steps on *each Agent in your cluster*. 
+
+*This guide assumes you already have [Mesos + Net-Modules](https://github.com/projectcalico/calico-docker/tree/master/docs/mesos#mesos--netmodules) installed on your Agent(s).* 
+
+> To obtain the Calico files, you will need `wget` installed. Please install the tool if you haven't already done so.
+
+## Download the Calico Mesos Plugin
+
+The Calico-Mesos plugin is available for download from the [calico-mesos repository releases](https://github.com/projectcalico/calico-mesos/releases). We recommend installing this binary in the `/calico` directory.
+
```

Comment:
I don't think we need to make any recommendation.  How about "In this example, we install the binary in the `/calico` directory."